devtools/projects/ng-devtools/src/lib/devtools-tabs/injector-tree/injector-providers.component.ts

```diff
@@ -42,8 +42,8 @@ import {Events, MessageBus, SerializedInjector, SerializedProviderRecord} from '
           }
         </mat-select>
       </mat-form-field>
-      @if (providers.length > 0) {
-      <table mat-table [dataSource]="providers" class="mat-elevation-z4">
+      @if (providers().length > 0) {
```

Comment:
While reviewing this PR I noticed this line and the line below it have a bug (not introduced in this PR). Can we update this to:

```suggestion
      @if (visibleProviders().length > 0) {
```

Reply:
Done